ui/apps/platform/src/services/DeferralConfigService.ts

```diff
@@ -0,0 +1,31 @@
+import axios from './instance';
+
+const vulnBaseUrl = '/v1/config/private/deferral/vulnerabilities';
+
+export type VulnerabilitiesDeferralConfig = {
+    expiryOptions: {
+        dayOptions: {
+            numDays: number;
+            enabled: boolean;
+        }[];
+        fixableCveOptions: {
+            allFixable: boolean;
+            anyFixable: boolean;
+        };
+        customDate: boolean;
+    };
+};
+
+export function fetchVulnerabilitiesDeferralConfig(): Promise<VulnerabilitiesDeferralConfig | null> {
```

Comment:
Can you remind me (in case we discussed it previously), what is returned as the initial config object? Is it `null`? I was wondering if we really needed to account for that here.

Reply:
Currently the initial config object is `null`, but Mandar mentioned that this would be changes before release. Once the BE change is in I'll update the service call here so that these types are more accurate.